dstk-infra/apollo/src/graphql/model-version/modelVersionMutations.ts

```diff
@@ -112,70 +106,86 @@ export const PresignURLForModelVersion = extendType({
         t.field('presignURL', {
             type: PresignedURL,
             args: { data: PresignedURLInputType },
+            async resolve(root, args, ctx) {
+                const modelStorageProvider = (await ObjectionMLModelVersion.relatedQuery(
```

Comment:
Today I learned `relatedQuery` was a thing, that's pretty neat lol

Reply:
Makes life sooooo nice in cases like these, minus the fact it makes typing go a little funny